scout/models/variant/variant.py

```diff
@@ -280,6 +78,17 @@ class Variant(Document):
   # Predicted deleteriousness:
   cadd_score = FloatField()
   clnsig = IntField()
+  
+  @property
+  def has_comments(self):
```

Comment:
I will need to remember to update my code to use this!

Should I push to this branch or handle it separately?